L34_extensions_in_M.ipynb

```diff
@@ -0,0 +1,548 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Perfect central extensions of $L_3(4)$ in the Monster group</Heading>\n",
+    "\n",
+    "There was [the question in MathOverflow](https://mathoverflow.net/questions/450255)\n",
+    "which perfect central extensions of the simple group $G = L_3(4)$ are subgroups of the sporadic simple Monster group $M$.\n",
+    "\n",
+    "The following computations with [OSCAR](https://oscar.computeralgebra.de) answer this question except for one candidate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " -----    -----    -----      -      -----   \n",
+      "|     |  |     |  |     |    | |    |     |  \n",
+      "|     |  |        |         |   |   |     |  \n",
+      "|     |   -----   |        |     |  |-----   \n",
+      "|     |        |  |        |-----|  |   |    \n",
+      "|     |  |     |  |     |  |     |  |    |   \n",
+      " -----    -----    -----   -     -  -     -  \n",
+      "\n",
+      "...combining (and extending) ANTIC, GAP, Polymake and Singular\n",
+      "Version\u001b[32m 0.13.0-DEV \u001b[39m... \n",
+      " ... which comes with absolutely no warranty whatsoever\n",
+      "Type: '?Oscar' for more information\n",
+      "(c) 2019-2023 by The OSCAR Development Team\n"
+     ]
+    }
+   ],
+   "source": [
+    "using Oscar;"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we get the list of perfect central extensions of $G$ (asuming that their character tables are contained in the character table library)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "14-element Vector{String}:\n",
+       " \"L3(4)\"\n",
+       " \"2.L3(4)\"\n",
+       " \"3.L3(4)\"\n",
+       " \"2^2.L3(4)\"\n",
+       " \"4_1.L3(4)\"\n",
+       " \"4_2.L3(4)\"\n",
+       " \"6.L3(4)\"\n",
+       " \"(2x4).L3(4)\"\n",
+       " \"(2^2x3).L3(4)\"\n",
+       " \"12_1.L3(4)\"\n",
+       " \"12_2.L3(4)\"\n",
+       " \"4^2.L3(4)\"\n",
+       " \"(2x12).L3(4)\"\n",
+       " \"(4^2x3).L3(4)\""
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "simp = character_table(\"L3(4)\");\n",
+    "extnames = all_character_table_names(is_perfect => true);\n",
+    "extnames = filter(x -> endswith(x, \"L3(4)\"), extnames);\n",
```

Comment:
Could also do this
```suggestion
    "filter!(x -> endswith(x, \"L3(4)\"), extnames);\n",
```

Reply:
Thanks for the hint (holds also for other places in the code).